modules/hcp-ibmz-dns.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-deploy/hcp-deploy-ibmz.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="hcp-ibmz-dns_{context}"]
+= DNS configuration for {hcp} on {ibm-z-title}
+
+The API server for the hosted cluster is exposed as a 'NodePort' service. A DNS entry must exist for the `api.<hosted_cluster_name>.<base_domain>` that points to the destination where the API server is reachable.
+
+The DNS entry can be as simple as a record that points to one of the nodes in the managed cluster that is running the hosted control plane.
+
+The entry can also point to a load balancer deployed to redirect incoming traffic to the Ingress pods.
+
+See the following example of a DNS configuration:
+
+[source,terminal]
+----
+$ cat /var/named/<example.krnl.es.zone>
```

Comment:
This style of user-replaceable is a bit non-standards (e.g., has periods and actual example-ish values).

Reply:
Point taken. This text came from a writer at IBM. I can check in with her to try to get a more standard example.